vhost-device-gpu/src/lib.rs

```diff
@@ -79,24 +81,33 @@ bitflags! {
 
 impl Display for GpuCapset {
     fn fmt(&self, f: &mut Formatter<'_>) -> std::fmt::Result {
+        if self.is_empty() {
+            return write!(f, "none");
+        }
+
         let mut first = true;
+        #[allow(unused_variables)]
```

Comment:
Is this still needed?

Reply:
not really, I changed it to `#[allow(unused_assignments)]` since building the null backend doesn't use the assignment.